### ISSUE TITLE
Console: Firmware update always require to always select a file before updating STM

### ISSUE THREAD

**Issue by switanis** [swift-nav/piksi_tools#269]
Console v0.26. 
Before pressing on Update STM button it is always required to browse and select a file. If the file is selected once and it has changed in the background (eg. after new compilation) Console will use the previously selected file and not the new one as expected.

**Comment by ljbade** [comment]
Yeah I have noticed this before. I always reselect the file each time to make sure it gets the new version.

**Comment by ljbade** [comment]
@denniszollo Would this be an easy fix? Always read the file?

**Comment by denniszollo** [comment]
Not an easy fix, but we can consider.  IMO the fix is to clear the file from memory in the console as soon as you bootload so at least it's not confusing.

**Comment by ljbade** [comment]
Yes that would also work
